browser/ui/views/playlist/playlist_action_bubble_view.cc

```diff
@@ -359,33 +423,55 @@ AddBubble::AddBubble(Browser* browser,
       l10n_util::GetStringUTF16(IDS_PLAYLIST_MEDIA_FOUND_IN_THIS_PAGE)));
   header->SetHorizontalAlignment(gfx::ALIGN_LEFT);
 
-  auto* scroll_view = AddChildView(std::make_unique<views::ScrollView>());
-  scroll_view->ClipHeightTo(/*min_height=*/0, /*max_height=*/230);
-  scroll_view->SetDrawOverflowIndicator(false);
-  scroll_view->SetBorder(views::CreateThemedRoundedRectBorder(
+  scroll_view_ = AddChildView(std::make_unique<views::ScrollView>());
+  scroll_view_->ClipHeightTo(/*min_height=*/0, /*max_height=*/230);
+  scroll_view_->SetDrawOverflowIndicator(false);
+  scroll_view_->SetBorder(views::CreateThemedRoundedRectBorder(
       /*thickness=*/1,
       /*corner_radius=*/4.f, kColorBravePlaylistListBorder));
-
-  list_view_ = scroll_view->SetContents(std::make_unique<SelectableItemsView>(
-      thumbnail_provider_.get(), items,
-      base::BindRepeating(&AddBubble::OnSelectionChanged,
-                          base::Unretained(this))));
-  list_view_->SetSelected(items);
+  scroll_view_->SetVisible(false);
+  loading_spinner_ = AddChildView(std::make_unique<LoadingSpinner>());
 
   // Fix preferred width. This is for ignoring insets that could be added by
   // border.
-  static constexpr int kWidth = 288;
-  scroll_view->SetPreferredSize(
-      gfx::Size(kWidth, scroll_view->GetPreferredSize().height()));
-
   SetButtonLabel(ui::DialogButton::DIALOG_BUTTON_OK,
                  l10n_util::GetStringUTF16(IDS_PLAYLIST_ADD_SELECTED));
 
+  if (playlist_tab_helper_->ShouldRefetch()) {
```

Comment:
Do you think `ShouldExtractMediaFromBackgroundWebContents` would be a better name?